src/core_codemods/subprocess_shell_false.py

```diff
@@ -0,0 +1,31 @@
+import libcst as cst
+from libcst import matchers
+from codemodder.codemods.api import BaseCodemod, ReviewGuidance
+from codemodder.codemods.utils_mixin import NameResolutionMixin
+from codemodder.codemods.api.helpers import NewArg
+
+
+class SubprocessShellFalse(BaseCodemod, NameResolutionMixin):
+    NAME = "subprocess-shell-false"
+    SUMMARY = "Use `shell=False` in `subprocess` Function Calls"
+    REVIEW_GUIDANCE = ReviewGuidance.MERGE_AFTER_CURSORY_REVIEW
+    DESCRIPTION = "Set `shell` keyword argument to `False`"
+    REFERENCES: list = []
```

Comment:
Please add the following references:
* https://docs.python.org/3/library/subprocess.html#security-considerations
* https://en.wikipedia.org/wiki/Code_injection#Shell_injection
* https://stackoverflow.com/a/3172488